Sources/Parley/Views/ParleyView.swift

```diff
@@ -499,8 +499,12 @@ extension ParleyView: ParleyDelegate {
     func didChangeState(_ state: Parley.State) {
         debugPrint("ParleyViewDelegate.didChangeState:: \(state)")
 
+        if state == .unconfigured {
```

Comment:
@alexkok This is the main solution for the issue.

Reply:
Interesting, I guess it isn't solved if we make the `ParleyView.deinit` method put the pollingService to `nil`, correct?

Reply:
No, indeed. On init the shared instance of Parley has still state configured and an accompanying messagesManager. It would be good to enforce in some sort of way that this state could not occur.